messenger/decoder.go

```diff
@@ -523,6 +524,19 @@ func readBodyState(d *httpDecoder) httpState {
 	return readBodyState
 }
 
+func isGracefulTermSignal(err error) bool {
```

Comment:
That's about as graceful as libprocess gets

On Thu, Sep 24, 2015 at 5:18 AM, Tomás Senart <notifications@github.com>
wrote:

> In messenger/decoder.go
> <https://github.com/mesos/mesos-go/pull/189#discussion_r40299426>:
>
> > @@ -523,6 +524,19 @@ func readBodyState(d *httpDecoder) httpState {
> >  	return readBodyState
> >  }
> >
> > +func isGracefulTermSignal(err error) bool {
>
> Is this name appropriate? AFAIK ECONNRESET doesn't mean the connection
> was *gracefully* closed on the other side, only that it was closed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/mesos-go/pull/189/files#r40299426>.
>